plugins/flowdock/lib/samson_flowdock/samson_plugin.rb

```diff
@@ -20,22 +21,47 @@ def flowdock_tokens
     def no_flowdock_token?(flowdock_attrs)
       flowdock_attrs['token'].blank?
     end
+
+    def enabled_flows_names
+      flowdock_flows.enabled.map(&:name)
+    end
+  end
+end
+
+Samson::Hooks.callback :deploy_defined do
+  Deploy.class_eval do
```

Comment:
OK. I'm not sure if it is worth the complexity since `Deploy` will always be defined when the Rails application loads. If we are going to have this hook we should probably have it when all models have been required rather than special casing `Deploy`.